lib/gds_api/email_alert_api.rb

```diff
@@ -77,6 +77,15 @@ def unsubscribe(uuid)
     post_json("#{endpoint}/unsubscribe/#{uuid}")
   end
 
+  # Unsubscribe from everything
+  # #
+  # @param integer Subscriber id
+  #
+  # @return [Hash] deleted
```

Comment:
Should this be returning a hash? Not really sure what it does return for a 204 maybe it's an empty object :-/

Reply:
I've marked it as `null` since that seems to be the closest thing